MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,423 @@
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            TextColor = Colors.Red,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Selected segment border color customization
+You can customize the selected segment border color, by using `Stroke` property of `SelectionIndicatorSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.SelectionIndicatorSettings>
+            <buttons:SelectionIndicatorSettings 
+                Stroke="Green"/>
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            Stroke = Colors.Green,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Selected segment border thickness customization
+You can customize the selected segment border thickness, by using `StrokeThickness` property of `SelectionIndicatorSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.SelectionIndicatorSettings>
+            <buttons:SelectionIndicatorSettings 
+                StrokeThickness="1"/>
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            StrokeThickness = 1,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Selection changed event
+The `SelectionChanged` event is triggered, once the selection process has been completed for the selected item in the Segmented Control. The `SelectionChangedEventArgs` has the following values, which provides information for `SelectionChanged` event:
```

Comment:
once the segment is selected in the Segmented Control

Reply:
Changed content